### DISCUSSION TITLE
Chernozems

### DISCUSSION THREAD

**Discussion by StephanMantelISRIC** [iuss-wrb/wrb#20] [Reference Soil Groups]
This discussion is to document the RSG Chernozems and document examples from sites and soils of Chernozems with identified diagnostics and qualifiers

**Comment by VolodymyrYakovenko2025** [comment]
Colleagues, greetings!
I propose for discussion the following additions to the classification of Chernozems and some other Reference Soil Groups (abstract of the article accepted for publication https://doi.org/10.1016/j.pedsph.2025.06.002):

Bioturbation caused by burrowing mammals is a geographically widespread factor in soil formation that can significantly influence soil morphology, properties, and dynamics. However, the current version of the World Reference Base for Soil Resources (WRB) does not offer a mechanism for adequately classifying pedons altered by mammalian bioturbation. Based on an analysis of the morphology of genetic soil profiles of Chernozems in the steppe zone of Ukraine, which have been transformed by bioturbation from Spalax microphthalmus, as well as the diversity of krotovinas and the micromorphological characteristics of their infill material, we propose the introduction of the qualifier "Krotovik" as a diagnostic criterion at the second level of WRB classification.
In accordance with the nomenclature adopted in the WRB classification system, we propose the following description of the qualifier Krotovik (kr) (from Slavic languages, e.g. Ukrainian 'krotovina', Polish 'kretowina', mole tunnel): having ≥ 30% (by volume, weighted average) filled and empty tunnels of burrowing mammals in the upper 100 cm of the mineral soil or to a limiting layer, whichever is shallower. In our opinion, a quantitative indicator of ≥ 30% is sufficient to apply the Krotovik qualifier in the classification of pedones transformed by mammalian bioturbation. In the list of qualifiers, Krotovik should be placed after Cambic, as the diagnostic value of the Skeletic, Vermic, Tonguic and Haplic qualifiers can fundamentally change as a result of mammalian bioturbation.
We recommend including Krotovik as one of the principal qualifiers in instances where mammalian bioturbation is a significant pedogenetic process that considerably modifies the essential properties of the soil. In the soil cover of Europe, this is particularly relevant to the Reference Soil Groups of Chernozems, Kastanozems, Phaeozems, Luvisols, and Cambisols.

**Comment by StephanMantelISRIC** [comment]
Dear Volodymyr,
Recently, the new correlation was made between Ukrainian and WRB classifications that allowed drawing a new WRB-based soil map of Ukraine. It will be published soon in the Soil Science Annual.
The proposal can be a result of discussions around the correlation table.
However, we already have a similar qualifier in WRB - Vermic. Although the name refers to primarily to earthworms, the definition includes all animal holes/casts/burrows.
All the best,
Cezary Kabala

**Comment by VolodymyrYakovenko2025** [comment]
Dear Cezary,

Thank you for your attention and for responding to my proposal.

I would like to offer some additional arguments in support of introducing the qualifier Krotovic.  
The differences between mammalian bioturbations and earthworm bioturbation are generally well known and well studied. In my view, these differences are fundamental for distinguishing their manifestations in soil research and classification.

1) Mammalian bioturbations and earthworm bioturbation differ significantly as processes:
- Earthworm bioturbation involves the gradual "percolation/gradual translocation" of material from one part of the soil profile to another in the form of individual earthworm casts, whereas mammals rapidly displace substantial volumes of soil material.
- As a result of earthworm bioturbation, structural aggregates are formed as earthworm casts with specific properties, while the burrowing activity of mammals mechanically fragments the soil into aggregates.
- There are also clear differences in the structure and properties of burrow networks created by mammals compared to those formed by earthworms.

2) Spatial expression of bioturbation in the soil profile and its impact on soil properties:
- Intensive earthworm bioturbation (which justifies the use of the qualifier Vermic) leads to a homogenized A horizon with uniform morphological, physical, and chemical properties.
- In contrast, intensive mammalian bioturbation disrupts the "typical" profile morphology and results in heterogeneity of physical (e.g., aggregate composition, bulk density) and chemical (e.g., distribution of carbonates and SOC) properties throughout the whole profile.

3) Geography of bioturbation in Chernozems:
- The arid climate limits earthworm populations in the steppe zone of Ukraine, and thus the intensity of earthworm bioturbation is insufficient to justify the use of the Vermic qualifier.
- Meanwhile, the distribution of burrowing mammals is not subject to such climatic constraints, making mammalian bioturbation more widespread geographically than earthworm bioturbation. This could likely be true for other regions of the world as well.

It is evident that mammalian bioturbations and earthworm bioturbation differ in terms of process mechanics, their impact on soil properties, and their geographic distribution. The only commonality is that both mammals and earthworms belong to the animal kingdom. Therefore, applying the Vermicqualifier to soils with sparse or absent earthworm populations—where intensive mammalian burrowing activity is observed—is not logical. In such cases, the use of Vermic qualifierrequires additional commentary in the soil classification description, which in practice often leads to the exclusion of the pedons transformed by mammalian bioturbation as “atypical” ones.

Sincerely,  
VolodymyrYakovenko

**Reply by StephanMantelISRIC** [reply]
Dear Volodymyr Yakovenko, 

Thank you for the interesting reflection on the bioturbation. You are right in the sense that the disruption of the layers and the vertical changes in the profile are more pronounced with the movement of moles and perhaps also that the geographies of the impact of bioturbation by worms and mammals are different, but this also counts for leaf cutting ants and termites that disrupt whole morphology and vertical gradients of soil profiles. It is tempting to add qualifiers for all conditions, and some are in favor of that, but in my opinion adding yet more qualifiers and diagnostics to the WRB system will increase. I have forwarded you message to Cezary by mail so he will reply to your message too.

Kid regards, Stephan Mantel
